repo/packages/P/postgresql-admin/0/marathon.json.mustache

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "{{service.name}}",
+    "cpus": {{postgresql-admin.cpus}},
+    "mem": {{postgresql-admin.mem}},
+    "instances": 1,
+    "env": {
+        "DB_HOST": "{{networking.postgres_host}}.marathon.l4lb.thisdcos.directory",
+        "DB_PORT": "{{networking.postgres_host_port}}"
+    },
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.postgresql-admin-docker}}",
+            "forcePullImage": false,
+            "network": "BRIDGE",
+            "portMappings": [
+            {
+                "containerPort": 80,
+                "hostPort": 0,
+                "servicePort": {{networking.port_external}}
+            }
+            ]
+        }
+    },
+    "healthChecks": [
+        {
+            "protocol": "HTTP",
+            "path": "/",
+            "portIndex": 0,    
+            "gracePeriodSeconds": 300,
+            "intervalSeconds": 60,
+            "timeoutSeconds": 20,
+            "maxConsecutiveFailures": 3
+        }
+    ],
+    "labels": {
+        "DCOS_SERVICE_NAME": "{{service.name}}",
+        "DCOS_SERVICE_SCHEME": "http",
+        "DCOS_SERVICE_PORT_INDEX": "0",
+        "DCOS_PACKAGE_NAME": "{{postgresql-admin.name}}",
+        "DCOS_PACKAGE_VERSION": "0.1",
+        "DCOS_PACKAGE_IS_FRAMEWORK": "false",
+        "HAPROXY_GROUP": "external",
```

Comment:
configurable and default to false. 

Vish was suggesting that in some cases we may want to leave apps outside admin router -- I think it's a good idea. Also, some customers will consider the DC/OS interface to be "Dan only"